lib/pathfinder/CGPathNode.h

```diff
@@ -37,6 +37,7 @@ enum class EPathAccessibility : ui8
 	NOT_SET,
 	ACCESSIBLE, //tile can be entered and passed
 	VISITABLE, //tile can be entered as the last tile in path
+	GUARDED,  //visitable, but in zone of control of nearby monster
```

Comment:
GUARDED_VISITABLE?

Reply:
Corrected comment.
GUARDED is used always when there is an active zone of control, whether there is a visitable object or not.